compute/wps/backends/backend.py

```diff
@@ -39,6 +41,28 @@ class Backend(object):
     def __init__(self):
         self.processes = []
 
+    def load_user(self, user_id):
+        try:
+            user = models.User.objects.get(pk=user_id)
+        except models.User.DoesNotExist:
+            raise WPSError('User "{}" does not exist', user_id)
+
+        return user
+
+    def load_job(self, job_id):
+        try:
+            job = models.Job.objects.get(pk=job_id)
+        except models.Job.DoesNotExist:
+            raise WPSError('Job {} does not exist', job_id)
+        
+        return job
+
+    def generate_output_path(self, user, filename):
+        job_id = user.job_set.count()
+
+        return '{}/{}/{}/{}.nc'.format(settings.WPS_PUBLIC_PATH, user.id,
```

Comment:
Would probably be better to use os.path.join

Reply:
Good catch, i'll have to change this in a new pr.